resources/js/components/docs/CombiDemoCodePreview.tsx

```
@@ -0,0 +1,87 @@
+                {isLoading && (
+                  <div className=" absolute right-0 left-0">
+                    <LogoSpinner className="mx-auto" />
+                  </div>
+                )}
+                <iframe
+                  src={demoUrl}
+                  className="m-0 h-screen p-0 bg-transparent roundedd-md absolute left-0 right-0 max-w-full top-0 rounded-md w-screen "
+                  loading="lazy"
+                  style={{ height: '100%' }}
+                  onLoad={() => setIsLoading(false)}
+                />
```

Biome (2.1.2)

[error] Provide a title attribute when using iframe elements.

Screen readers rely on the title set on an iframe to describe the content being displayed.

(lint/a11y/useIframeTitle)